exercises/practice/eliuds-eggs/create_fixture.sql

```diff
@@ -0,0 +1,10 @@
+DROP TABLE IF EXISTS "eliuds-eggs";
+CREATE TABLE "eliuds-eggs" (
+    "number" INT,
+    "result" INT
+);
+
+-- Note: the CSV file contain literal tab, newline, carriage returns.
```

Comment:
```suggestion
-- Note: the CSV file contains literal tab, newline, carriage returns.
```

Reply:
"may contain". It doesn't actually contain those.